receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -30,21 +30,33 @@ var (
 	errEmptyMetricValue = errors.New("empty metric value")
 )
 
-func getSupportedTypes() []string {
-	return []string{"c", "g", "h", "ms"}
-}
+type (
+	MetricType   string // From the statsd line e.g., "c", "g", "h"
+	TypeName     string // How humans describe the MetricTypes ("counter", "gauge")
+	ObserverType string // How the server will aggregate histogram and timings ("gauge", "summary")
```

Comment:
I added more type names to help clarify all the bare strings in this code. The logic in (Config).validate() now uses these types, so they're exported.